extensions/wikibase/module/scripts/schema-alignment.js

```diff
@@ -433,8 +433,39 @@ SchemaAlignment._changesCleared = function() {
         .addClass('disabled');
 };
 
-SchemaAlignment._createDraggableColumn = function(name, reconciledSiteIRI) {
+SchemaAlignment._createDraggableColumn = function(name, reconciledSiteIRI, reconConfig) {
+  var serviceUrl = null;
+  var service = null;
+  var serviceLogo=null;
+  var urlIsValid=false;
+  if (reconConfig) {
+    serviceUrl =reconConfig.service;
+  }
+  try {
+    
+    if (new URL(serviceUrl)) {
```

Comment:
The URL that we should validate here is not the service URL, but the `service.logo` URL.